doc/source/examples/resolved-cfd-dem/cylinder-with-sharp-interface/cylinder-with-sharp-interface.rst

```diff
@@ -0,0 +1,172 @@
+==============================================================================
+Flow around a cylinder using the sharp interface method
+==============================================================================
+Features
+----------------------------------
+- Solvers: ``gls_sharp_navier_stokes_2d`` (with Q1-Q1) 
+- Steady-state problem
+- Explains how to set sharp interface immersed boundary to reprecent a particle
+- Specifies an initial condition
+- Displays the use of non-uniform mesh adaptation 
+
+Location of the example
+------------------------
+
+- Parameter file: ``/examples/incompressible_flow/2d_flow_around_cylinder/cylinder.prm``
+
+
+Description of the case
+-----------------------
+
+In this example, we study the flow around a static cylinder in 2D using the sharp-interface method to represent the cylinder. The geometry of the flow takes the same basic case defined in 
+`Example 3 <../../incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.html>`_. As such, we use the parameter file associated with `Example 3 <../../incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.html>`_ as the base for this example. The exact dimensions for this example can be found in the following figure. 
+
+.. image:: images/cylinder_case.png
+    :alt: Simulation schematic
+    :align: center
+
```

Comment:
I think we should add a Parameter file header hear and then subheaders for parameter subsections